server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -72,9 +80,17 @@ export default class PrintLicenceRoutes {
       user
     )
 
-    const additionalConditions = licence.additionalLicenceConditions.filter(
-      (c: AdditionalCondition) => !additionalConditionsWithUploads.find((c2: AdditionalCondition) => c.id === c2.id)
-    )
+    const getGroupedAdditionalConditions: Map<string, AdditionalCondition[]> =
+      this.getGroupedAdditionalConditions(licence)
+    const singleItemConditions: AdditionalCondition[] = []
+    const multipleItemConditions: AdditionalCondition[][] = []
+    getGroupedAdditionalConditions.forEach(value => {
+      if (value.length > 1) {
+        multipleItemConditions.push([...value])
+      } else {
+        singleItemConditions.push(...value)
+      }
+    })
```

Comment:
not sure if this is better
```
const singleItemConditions = groupedAdditionalConditions.filter(v => v.length == 1).flatMap(v => v)
const multiItemConditions = groupedAdditionalConditions.filter(v => v.length > 1)
```

Reply:
yes updated and moved this common code that was in renderPdf and preview methods into another method